hub-saml/src/main/java/uk/gov/ida/saml/hub/transformers/inbound/providers/DecoratedSamlResponseToIdaResponseIssuedByIdpTransformer.java

```diff
@@ -1,8 +1,8 @@
 package uk.gov.ida.saml.hub.transformers.inbound.providers;
 
 import org.opensaml.saml.saml2.core.Response;
+import uk.gov.ida.eidas.logging.EidasAttributesLogger;
```

Comment:
I've just noticed that it feels like maybe this class is not in the right namespace if it's coming from samllibs?

Reply:
What makes you say that - what would you suggest otherwise?

Reply:
I think the namespace is fine - we already have a bunch of namespaces in saml-libs that are....there

Reply:
Although admittedly `matchingserviceadapter` and `verifyserviceprovider` shouldn't be namespace names in a generic library. We can refactor this later, though

Reply:
Yeah, I think if it's in a shared library, then by definition we shouldn't have classes that are specific to either eIDAS, or to the MSA, or the VSP – and the namespaces would reflect that.

Reply:
Totally agree, George 👍